src/ast/ast.h

```diff
@@ -377,11 +377,11 @@ class AssignVarStatement : public Statement {
 class AssignConfigVarStatement : public Statement {
 public:
   AssignConfigVarStatement(Diagnostics &d,
-                           std::string config_var,
+                           Identifier *config_var,
                            Expression *expr,
                            Location &&loc);
 
-  std::string config_var;
+  Identifier *config_var = nullptr;
```

Comment:
Just as aside: we do have a weird habit of providing default values for these things, despite them not being optional nor having a default constructor. I'm not saying you should do anything different here, but it's been on my list for a long time because it creates ambiguity.